tools/Azure.Mcp.Tools.Redis/src/Models/Resource.cs

```diff
@@ -0,0 +1,109 @@
+using Azure.Mcp.Tools.Redis.Models.ManagedRedis;
+
+namespace Azure.Mcp.Tools.Redis.Models
+{
+    public class Resource
+    {
+        /// <summary> Name of the Redis cache resource. </summary>
+        public string? Name { get; set; }
+
+        /// <summary> Type of Redis cache, either AzureCacheForRedis or AzureManagedRedis. </summary>
+        public string? Type { get; set; }
+
+        /// <summary> ID of the Azure subscription containing the Redis cache resource. </summary>
+        public string? SubscriptionId { get; set; }
+
+        /// <summary> Name of the resource group containing the Redis cache resource. </summary>
+        public string? ResourceGroupName { get; set; }
+
+        /// <summary> Azure geo-location where the Redis cache resource lives. </summary>
+        public string? Location { get; set; }
+
+        /// <summary> SKU of the Redis cache resource. </summary>
+        public string? Sku { get; set; }
+
+        /// <summary> Provisioning status of the Redis cache resource. </summary>
+        public string? ProvisioningState { get; set; }
+
+        /// <summary> Version of Redis server supported by the cache. </summary>
+        public string? RedisVersion { get; set; }
+
+        /// <summary> DNS host name clients use to connect to the Redis cache. </summary>
+        public string? HostName { get; set; }
+
+        /// <summary> Minimum version of TLS supported for client connections to this Redis cache. </summary>
+        public string? MinimumTlsVersion { get; set; }
+
+        /// <summary> Resource IDs of private links used for network-isolated client connections to the Redis cache. </summary>
+        public string[]? PrivateEndpointConnections { get; set; }
+
+        /// <summary> The availability zones in which the Redis cache is deployed. </summary>
+        public string[]? Zones { get; set; }
+
+        /// <summary> System-assigned managed identity of the Redis cache resource. </summary>
+        public ManagedIdentityInfo? Identity { get; set; }
+
+        /// <summary> Tags on the Redis cache resource. </summary>
+        public IDictionary<string, string>? Tags { get; set; }
+
+        /// *****************************************
+        /// Azure Managed Redis-Specific Properties
+        /// *****************************************
+
+        /// <summary> Current status of the Azure Managed Redis or Azure Redis Enterprise resource. </summary>
+        public string? ResourceState { get; set; }
+
+        /// <summary> Databases in the specified Azure Managed Redis or Azure Redis Enterprise resource. </summary>
+        public Database[]? Databases { get; set; }
+
+        /// *****************************************
+        /// Azure Cache for Redis-Specific Properties
+        /// *****************************************
+
+        /// <summary> Port for TLS (aka SSL) client connections to the Azure Cache for Redis resource. </summary>
```

Comment:
```suggestion
        /// <summary> Port for TLS (aka SSL) client connections to the Redis resource. </summary>
```
(remove "Azure Cache for " from all the comments on these OSS-specific properties)